xcbgen-rs/src/defs/mod.rs

```diff
@@ -333,6 +338,38 @@ impl Namespace {
             })
             .cloned()
     }
+
+    /// Tell if the namespace contains file descriptors, or depend on any
+    /// modules containing file descriptors.
+    pub fn contains_fds(&self) -> bool {
```

Comment:
Isn't this function and the `contains_fds` field now unused? At least I can't find any callers of it.